changelog.md

```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Add action to redirect on user page.
   * Add action to delete a user.
 * Setup User page with:
-  * Display all information of a user.
+  * Display all information and groups of a user.
```

Comment:
make two sentence because we will have more functionnality

Reply:
done